target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl838x-sram.S

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Realtek RTL838X SRAM clock setters
+ * Copyright (C) 2022 Markus Stockhausen
+ *
+ * relocatable code that will run from SRAM
```

Comment:
i'm a sucker for understanding the purpose in these headers. 'Why do we need this? (besides the what)